checkout-v3/index.md

```diff
@@ -217,6 +218,7 @@ implementation.
   %}
 
 [apple-pay-logo]:/assets/img/applepay-logo.svg
+[c2p-logo]: /assets/img/clicktopay-logo.svg
```

Comment:
Does `/assets/img/clicktopay-logo.svg` already exist or did you forget to add it in this PR?